pyClient/zethMock.py

```diff
@@ -4,104 +4,47 @@
-/j2XqiV4HNhkql39vt76oy6RV7Yl3KIW+dsT5EwZos8NmgvWo28pC4u+4nXbuNLH
-WVVt1jHQVhG9EQRlbkoCypDD4wOmrdlJplCjaRgCSeN8U7G+MTr2AtRT+0VozV04
-mIoKPDymx+pgH8KJVQIDAQAB
------END PUBLIC KEY-----"""
-    BobDecKey = """-----BEGIN RSA PRIVATE KEY-----
-MIICXAIBAAKBgQC3Ba45mM+JhO9tNpHwldnnvAtA/j2XqiV4HNhkql39vt76oy6R
-V7Yl3KIW+dsT5EwZos8NmgvWo28pC4u+4nXbuNLHWVVt1jHQVhG9EQRlbkoCypDD
-4wOmrdlJplCjaRgCSeN8U7G+MTr2AtRT+0VozV04mIoKPDymx+pgH8KJVQIDAQAB
-AoGBAJ7wNvHby3cgU5AjUK9+YuKEgb1qTHC2GJ3rZtxcuw0NwbQlG96qLgtJRBXx
-2xe2LYQhx++G9HrsKS+a0Dvvi+rQ7YK7cxFJuKNmwonoKI9LpIFDV7xvNJ1TPU1G
-QVskoU1OfwUabyDmYI5j7Lgf2xqu7Z2xNz1iQoMzFvI+ffItAkEA2GY+N3E3ZJWP
-+E6OuZHD08E0h9mPGoug5YQjhYEV0zyI9abGCzDZp90YKoLerTJefaD5NyErRVFn
-7jnFYGjxewJBANiDzSTMUeojGWboRhfHl/2FNSjg7ClgI8tjdH72Rn46EbLJHgno
-L3j5O1XwwgtbjTb/vRpc0V+gQp32qSlw728CQAhPnPIaKgt15wqdUcP0wjWexPq2
-s1VMqYhHE+ors//h4ky09AQ4AxP8XNI9Jno2ZgSjKw8f+f52iuxOUbNLNIMCQFfs
-vkQxTRqeAlTOApjpjwl/LOVa4cyzpBWWX9qnPF1KS6GlFrPDPHQOElCGIubl2OT6
-2dp40vXYaPUpE+0mVbUCQCxw7IvglwRKc142K7HfsSOdn0bQplS3ezEthriIzacP
-CCePPHuHI3A7+3ROFMKXmmjDauEMcpLhQen5f4/Corg=
------END RSA PRIVATE KEY-----"""
+    Bob25519PublicKey = b'2cb62682cb3f12bc320c7fa37caf14da344383bc980de3ac43c635237b725b2a'
+    Bob25519PrivateKey = b'ec6e45c26de33403c9e4059efdc940939b85f3b340dd95b7a9e32e24601960b6'
 
     # Charlie credentials in the zeth abstraction
     CharlieOwnershipKeys = zethGRPC.generateApkAskKeypair()
-    CharlieEncKey = """-----BEGIN PUBLIC KEY-----
-MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDz6F8PhRiHVCnfq5jxOx+N8Uso
-v35NJSWQ3R/iRmNK+BeNedXbqvunEbLPEdus5h9BE2RwR0wumDe7WJWIjjRLEU7C
-5dJGDEviWlJBC+yw0wbnWA5FV6Mrq0UJSLVe5Q5uiLuHuzI9Ag9UOqJZTXQ5yfG8
-9QRE8HumA1tzfxCrQwIDAQAB
------END PUBLIC KEY-----"""
-    CharlieDecKey = """-----BEGIN RSA PRIVATE KEY-----
-MIICWwIBAAKBgQDz6F8PhRiHVCnfq5jxOx+N8Usov35NJSWQ3R/iRmNK+BeNedXb
-qvunEbLPEdus5h9BE2RwR0wumDe7WJWIjjRLEU7C5dJGDEviWlJBC+yw0wbnWA5F
-V6Mrq0UJSLVe5Q5uiLuHuzI9Ag9UOqJZTXQ5yfG89QRE8HumA1tzfxCrQwIDAQAB
-AoGATQ6v4bZZ7n9Pj2OmOShFqtF9vkzpeTPwL1k89n7oZcoFnuPMBc96G+lChZsN
-vQ0i+KtIwxQzZFEg4mZ1L6RFrofOyveFxsdI3LdpAbIKZujfatnsIjvjfYuZtV71
-63oP9HAbQnatrx3vhzZgw+FoIp/0M14J3wmHC/GpdLGxABkCQQD32fzy0lSzqVTp
-elxw7U+Rkze9WPcUEQYyaFDiB4COTudFwVTxoBZd2vxfbxpfr03no59shslZxzcl
-GWPs2gbnAkEA++0wy+H7DSTSQaDFsciVId6qAvNfM9wEUpq1WepvZTvOhyULp9PI
-TrxCtvYKPjKqU/7rPsJ8eVbBoAlhDJXZRQJAeu3imKkbm7R7ygWHffcmBOUIu2A5
-w/khorS8kS75Yxvdd2qJcAJftZNcoxTe9uBi+mXcN56ulVnKjxsFxb7ptwJAWixi
-RPgURnYhlEAZwzMKvl7W98tpDkT4fyDFPPP+/3tSx2jpLR9PGW+laZvTusOj2ADs
-7z/qEfyNvdzdkgWpCQJAZyTN+aWuoqhR9h3fsCLy4NisJ3z5reViXZVMKW4J8jat
-aKtIj2rMlUbT+hLkAQmUb4YZwxtPibPTIUFTwrHmiw==
------END RSA PRIVATE KEY-----"""
+    Charlie25519PublicKey = b'a3058cdc22e4600de1b925ed02743a4310cb3011ec6a953abd9c9b859c9cec6d'
+    Charlie25519PrivateKey = b'bce6da6eda2bfe127dc1f3b7406d4f9f36d0747d04daa14b5a296fef54e0f56e'
 
     keystore = {
         "Alice": {
             "AddrPk": {
-                "ek": AliceEncKey,
+                "pubkey": Alice25519PublicKey,
```

Comment:
I would rename pubkey to enc_pubkey or something similar (likewise for privkey) to explicitly say the keys are for encryption.

Reply:
I was also evaluating if put it back as before as `ek` and `dk`

Reply:
I wouldn't call it ek/dk because they are not the true encryption and decryption keys (well not directly).

Reply:
Renamed. @rrtoledo resolve close the conversation if you are happy with that.